Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -683,7 +689,7 @@ def integration_health():
     })
 
 
-def validate_auth_header(headers: dict) -> bool:
+def validate_auth_header(headers: dict) -> bool:    # pragma: no cover
```

Comment:
Why did you add `# pragma: no cover` on this function?

Reply:
it was all red in the coverage report